components/modelmeshserving/modelmeshserving.go

```diff
@@ -107,9 +107,7 @@ func (m *ModelMeshServing) ReconcileComponent(ctx context.Context,
 
 		if err := cluster.UpdatePodSecurityRolebinding(ctx, cli, dscispec.ApplicationsNamespace,
 			"modelmesh",
-			"modelmesh-controller",
-			"odh-prometheus-operator",
-			"prometheus-custom"); err != nil {
```

Comment:
Is this something that was added for user workload monitoring?

Reply:
should not be, in odh 2.0.0 alread have these https://github.com/opendatahub-io/opendatahub-operator/blob/23299563e8e92e1c057fdf1cfac96cc8e6a7a3ff/components/modelmeshserving/modelmeshserving.go#L40

Reply:
@VedantMahabaleshwarkar if you know this ^?

Reply:
should be fine to remove `odh-prometheus-operator` and `prometheus-custom` IIRC. But it's been a while, if there is a way to ensure the SRE stack (for managed RHOAI) is working for modelmesh alerts with this change then it should be fine. I'm 90% certain these aren't needed for the SRE stack but it would be good to get that confirmation.

Reply:
we could  do a downstream build and let QE test on a managed cluster, see how it goes

Reply:
test result: https://issues.redhat.com/browse/RHOAIENG-13411?focusedId=25916040&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25916040